### ISSUE TITLE
Feature request: allow to change the font size

### ISSUE THREAD

**Issue by afiskon** [keybase/client#8638]
Hi,

The font size that is used by the desktop Keybase client is too small to my taste and I didn't find an option that allows to increase it. It's not a big problem, but it would be great to add such an option in the next release.

**Comment by zapu** [comment]
If you are on Windows 10, Keybase app respects desktop scaling.

This has been also asked for in https://github.com/keybase/client/issues/5797 and this proposal is tracked internally and might be added, but it's low priority. There is a workaround (mentioned in the issue linked) of using `--force-device-scale-factor=1.25` argument, but I've never tried it myself.

**Comment by afiskon** [comment]
> If you are on Windows 10, Keybase app respects desktop scaling.

No, I'm afraid I'm using Arch Linux.

**Comment by michaelvolz** [comment]
I am using Windows 10 with Desktopscaling but the font size is still too small. I really would love some kind of font size managemant or the possibilty to zoom the whole GUI like Chrome does, please.

**Comment by michaelvolz** [comment]
@zapu Your tipp to change the scale-factor using a commandline switch did the trick for me. Didn't understand exactly what you meant at first. Thanks. 

I have 1.25 default for my system and used `--force-device-scale-factor=150`
Much better now.

**Comment by zzeligg** [comment]
Where would that command line switch be added under MacOS? I seem to only have a Keybase.app file in my Applications folder.

**Comment by cfoster** [comment]
Same as #5797 and #8820

**Comment by jacksonrayhamilton** [comment]
@zapu please raise the priority on this feature request.  Not all of us have eagle vision.  Accessibility-first!

The command-line switch is a start but really unfriendly if we want to get non-tech people to communicate with us with these tools.

**Comment by jacksonrayhamilton** [comment]
BTW, I just started using these clients and am really enjoying them.  Would be happy to help contribute this improvement.  Would you have a specific direction in mind for this feature, or would you like me to figure it out myself?

**Comment by tstrijdhorst** [comment]
+1 I love the app but I have to really concentrate to read the small text (especially since the color scheme is also not that read friendly).

**Comment by egypt** [comment]
I disagree that this is not a big problem. On my 14" screen with a 2560x1440 native resolution, the text is unreadably small.

**Comment by jacksonrayhamilton** [comment]
Hey maintainers (@patrickxb / @maxtaco / @cjb), do you have a specific direction in mind for this feature, or would you like me to figure it out myself?

**Comment by denibertovic** [comment]
Is there any progress on this?

**Comment by jacksonrayhamilton** [comment]
There could be.  All the maintainers need to do is respond to us saying “yes, please work on it.”  That’s it!  What are they waiting for?  Free code coming your way, all you need to do is acknowledge that you would like this and will review it if we put it forward.

**Comment by l1x** [comment]
It seems Keybase guys do not care if you can read the text in their app or not. Lucky they do not have a chat application, it would suck.

**Comment by saintjames1911** [comment]
Keybase developers, please address this issue.  I tried changing Windows 10 scaling but the app does not respond to those changes.

**Comment by l1x** [comment]
For the Mac users you can use Command  + + / - to adjust the UI size. I just figured it out recently. This makes everything bigger but at least it is possible to read now the text. It is a good workaround.

**Comment by saintjames1911** [comment]
CMD +/- works in Windows 10 as I1x described.  To clarify, the "+" and "-" keys on the main keyboard cluster work... the numpad keys do not work.
